code/game/machinery/seed_extractor.dm

```diff
@@ -8,16 +8,38 @@
 
 /obj/structure/machinery/seed_extractor/attackby(obj/item/O as obj, mob/user as mob)
 
+		// Plant bag and other storage containers.
+	if(istype(O,/obj/item/storage))
```

Comment:
you can probably typecheck for plant bag here instead of storage in general, I doubt someone would use backpack for this

Reply:
I mean eh I could but why not leave it open for storage, chance like a mess tech dumps a box of stuff they get from req into their extractor